data/seed/Seeding.md

```diff
@@ -0,0 +1,180 @@
+# Seeding the Database
+
+This covers seeding the database for development work.
+
+## Preparation
+
+You should have followed the steps for getting started and have:
+
+- PostgreSQL database and Elasticsearch search engine running in Docker containers.
+- Vite front end server running in another terminal window.
+- Django development server running in another terminal window.
+
+At this point, you should be able to login to the Admin website at
+http://localhost:8000/ or http:127.0.0.1:8000/ with the email address and
+password of the super user you created.
+
+In a fourth terminal window, have your Python virtual environment activated and
+be in your working directory with the UIPA.org source files cloned from your
+fork of the main branch of the [UIPA.org
+repository](https://github.com/CodeWithAloha/uipa).
+
+## Seeding
+
+### Load Classifications, Jurisdictions, and Freedom of Information Laws
+
+Three fixtures will be used to load seed data that is not going to change very
+much for development work.
+
+This will set up the data that needs to exist in the database before you upload
+the public body information from a CSV file.
+
+```
+$ python manage.py loaddata data/seed/2024-03-15-classification.json
+$ python manage.py loaddata data/seed/2024-03-15-jurisdiction.json
+$ python manage.py loaddata data/seed/2024-03-15-foilaw.json
+```
+
+*NOTE:*
+
+> The Classifications data has been extracted from the Classifications field in
+> the extract of public bodies from the production UIPA.org website on
+> 03/15/2024. See 2024-03-15-Hawaii_UIPA_Public_Bodies_All.csv.
+
+
+### First Test Load of Public Bodies
+
+For the first time that you seed the database, you'll be using a very small
+subset of data to see how things go.
+
+- Run this command to load the category data that's needed to load the subset
+  of public body data:
+
+    ```
+    $ python manage.py loaddata data/seed/test-categories.json
+    ```
+
+- Then, on the `Public Body` page (Home > Public Body > Public Bodies), scroll
+  down to the bottom of the page to where there is a `Choose File` button next
+  to the `Import Public Bodies` button.
```

Comment:
Have you tried running this? 

`python manage.py import_csv data/seed/2024-03-24-public-bodies-fixed.csv`

Docs: https://froide.readthedocs.io/en/latest/importpublicbodies/#importing-via-command-line

I gave it a quick shot, and it seemed to work:

```
➜  uipa git:(pr/80) python manage.py import_csv data/seed/test-public-bodies.csv
/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django_elasticsearch_dsl/documents.py:178: ElasticsearchWarning: Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.15/security-minimal-setup.html to enable security.
  response = bulk(client=self._get_connection(), actions=actions, **kwargs)
Import done.
```